asf_heat_pump_affordability/pipeline/produce_costs_dataset.py

```diff
@@ -30,34 +30,44 @@ def run():
         type=int,
     )
 
+    parser.add_argument(
+        "--cpi_data_year",
+        help="Reference year to adjust heat pump installation costs to.",
+        type=int,
+    )
+
     args = parser.parse_args()
     main(**vars(args))
 
 
 def main(
-    mcs_epc_join_date: int, cost_year_min: int = None, cost_year_max: int = None
+    mcs_epc_join_date: int,
+    cpi_data_year: int,
+    cost_year_min: int = None,
```

Comment:
To properly type hint these kind of defaults with None, use something like:

from typing import Optional

cost_year_min: Optional[int] = None